src/main/fc/cli.c

```diff
@@ -1903,26 +1903,39 @@ static void cliServoMix(char *cmdline)
 
 #ifdef USE_PROGRAMMING_FRAMEWORK
 
-static void printLogic(uint8_t dumpMask, const logicCondition_t *logicConditions, const logicCondition_t *defaultLogicConditions)
+static void printLogic(uint8_t dumpMask, const logicCondition_t *logicConditions, const logicCondition_t *defaultLogicConditions, int32_t showLC)
```

Comment:
I dont think we need `int32_t` over here. `int16_t` should be enough. And because we have only 64 LCs, ever `int8_t` should do

Reply:
Agree. I changed the loop in the print function to use uint16. uint8 would be fine in there. But I would need to use uint16 for the passed in value to avoid compiler warnings.